tools/Crank/benchmarks.yml

```diff
@@ -21,6 +21,37 @@ scenarios:
       variables:
         path: /api/Hello
         duration: 180
+  http-appsvc:
+    application:
+      job: server
+      environmentVariables:
+        HOME: "{{ TempPath }}"
+        WEBSITE_SITE_NAME: "Test"
```

Comment:
These are the standard App Service env vars that the host inspects to determine whether it's running in App Service